mathics/core/expression.py

```diff
@@ -620,7 +597,9 @@ def elements(self):
 
     @elements.setter
     def elements(self, values: Iterable):
-        self._elements = self._build_elements(values)
+        self._elements = tuple(values)
+        # Set to build self.elements_properties on next evaluation()
+        self.elements_properties = None
```

Comment:
This doesn't seem like the right place.

Reply:
I am getting the sense that adding an element setter property was a good idea in the first place. 
Right now though I don't want to go through the code to remove all of the callers.

Because elements is a setter, more than on argument generally is not passed. We also don't want to aggressively compute element properties here because we don't know if the expression that this is in will be evaluated and there can be a lot of elements.  

So the safest thing to do is to set this to `None` which you will find mentioned in docstrings as indicating that on the next  rewrite_apply_eval  step properties need to be computed.